SpatialGDK/Source/SpatialGDK/Private/LoadBalancing/LayeredLBStrategy.cpp

```diff
@@ -123,6 +126,13 @@ VirtualWorkerId ULayeredLBStrategy::WhoShouldHaveAuthority(const AActor& Actor)
 	return ReturnedWorkerId;
 }
 
+SpatialGDK::FActorLoadBalancingGroupId ULayeredLBStrategy::GetActorGroupId(const AActor& Actor) const
+{
+	const FName ActorLayerName = GetLayerNameForActor(Actor);
+
+	return LayerData.FindChecked(ActorLayerName).LayerIndex + 1;
```

Comment:
Should I add a comment here detailing this `+1`? It's just so that actor group IDs are non-zero when using layered strategies.
Also, could be fun if we used nested layered strategies.

Reply:
Yes I would add the comment.